testnet/stacks-node/src/neon_node.rs

```diff
@@ -2255,6 +2262,151 @@ impl BlockMinerThread {
         return false;
     }
 
+    /// Read any mock signatures from stackerdb and respond to them
+    pub fn send_mock_miner_message(&mut self) -> Result<(), ChainstateError> {
+        let miner_config = self.config.get_miner_config();
+        if !miner_config.pre_nakamoto_miner_messaging {
+            debug!("Pre-Nakamoto mock miner messaging is disabled");
+            return Ok(());
+        }
+
+        let burn_db_path = self.config.get_burn_db_file_path();
+        let burn_db = SortitionDB::open(&burn_db_path, false, self.burnchain.pox_constants.clone())
+            .expect("FATAL: could not open sortition DB");
+        let p2p_net = StacksNode::setup_peer_network(
```

Comment:
I don't think you need to go so far as to instantiate a whole other `PeerNetwork`.

Reply:
Yes this was a temp. have replaced!